src/libpmem2/deep_flush_linux.c

```diff
@@ -30,23 +30,44 @@ pmem2_deep_flush_write(unsigned region_id)
 
 	char deep_flush_path[PATH_MAX];
 	int deep_flush_fd;
+	char rbuf[2];
+	int rv = 0;
```

Comment:
we are not changing value of this varialbe, instead return rv; you can use return 0